Analyses/test/bfieldPlotter.fcl

```diff
@@ -0,0 +1,32 @@
+# -*- mode:tcl -*-
+# Example of running BFieldPlotter
+#
+
+#include "JobConfig/primary/flateminus.fcl"
+process_name : BFieldPlotter
+
+physics.analyzers.bfieldplotter : {
+    module_type : BFieldPlotter
+    plane       :  "x"
+    planeValue  :  -10000.
+    axisOneMin  : -8050.  # -8050. #-10500.
+    axisOneMax  :  8050.  #  8050. # 10500.
+    axisTwoMin  : -25000. # -8050. # -25000.
+    axisTwoMax  :  45000. #  8050. #  45000.
+    mapBinSize  :  10.
+}
+
+physics.e1             : [bfieldplotter]
+physics.end_paths      : [e1]
+physics.p1             : [generate, g4run]
+physics.trigger_paths  : [p1]
+
```

Comment:
You should not need either generate or g4run to make this work.  Get rid of them.

Actually I think that the only line that you actually need is:
physics.e1             : [bfieldplotter]

Reply:
Fixed this, thank you!